### ISSUE TITLE
[Website Request] https://mangafire.to/home

### ISSUE THREAD

**Issue by BelminH** [MALSync/MALSync#1699]
### Website name

Mangafire

### Website link

https://mangafire.to/home

### Website language(s)

EN/ FR/ ES-LA/ PT-BR

**Comment by BelminH** [comment]
@lolamtisch blocked?

**Comment by lolamtisch** [comment]
Yes. We will need wait until the page fixes it, before we can continue.
